.changelog/config.toml

```diff
@@ -0,0 +1 @@
+project_url = 'https://github.com/informalsystems/tendermint-rs'
```

Comment:
This change seems unrelated to the premise of this PR, could you contextualise what constitutes this change?

Reply:
It's a prerequisite for being able to automatically generate changelog entries from the CLI using unclog, as per https://github.com/informalsystems/unclog#adding-a-new-unreleased-entry

I could've probably made the change in a separate PR, but it's so small, and I had to do it anyways, that I thought I'd just add it in here.